packages/cashier/src/components/crypto-fiat-converter/__tests__/crypto-fiat-converter.spec.tsx

```diff
@@ -4,12 +4,13 @@ import CryptoFiatConverter from '../crypto-fiat-converter';
 import { Formik } from 'formik';
 import * as formik from 'formik';
 import CashierProviders from '../../../cashier-providers';
+import { mockStore, TStores } from '@deriv/stores';
```

Comment:
```suggestion
import { mockStore } from '@deriv/stores';
```
@rostislav-deriv Use `TRootStore` instead 🙏🏻